optaplanner-build-parent/pom.xml

```diff
@@ -730,6 +730,23 @@
                 <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
               </manifest>
             </archive>
+            <excludes>
+              <!-- Remove unnecessary metadata generated by the jaxb2-maven-plugin. -->
+              <exclude>META-INF/JAXB/</exclude>
+              <!-- The XSD should be available only on the https://www.optaplanner.org. -->
+              <exclude>optaplanner.xsd</exclude>
```

Comment:
Doesn't the jaxb maven plugin have a property to not put those files in the jar file?

Reply:
No; it can be moved to a different location, though, but it needs to be on the classpath so that we can test it.